app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -651,7 +651,11 @@ public void find_new_raw_curve() {
             BgReading latest_entry = BgReading.lastNoSenssor();
             ra = 0;
             rb = 0;
-            rc = latest_entry.age_adjusted_raw_value;
+            if (latest_entry != null) {
```

Comment:
Im struggling to see how this could have ever been null (This gets called just after bgReading.save no? are we calling it from somewhere else now??

Or did we break the lastNoSenssor query?

OR are we creating bg engries without having a sensor (which would be bad and will break a lot of things)